test/core/end2end/fuzzers/BUILD

```diff
@@ -12,31 +12,51 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("//bazel:grpc_build_system.bzl", "grpc_package")
+load("//bazel:grpc_build_system.bzl", "grpc_cc_library", "grpc_package", "grpc_proto_library")
 load("//test/core/util:grpc_fuzzer.bzl", "grpc_fuzzer", "grpc_proto_fuzzer")
 
 grpc_package(name = "test/core/end2end/fuzzers")
 
 licenses(["notice"])
 
+grpc_proto_library(
```

Comment:
What's the argument for a single top level message across disparate fuzzers vs the proposed different top-level fuzzer message per fuzzer that depends-on the api-fuzzer method bits?

Reply:
I think it will be cleaner to just encode the different top level fuzzer message into a separate fuzzer action and have different fuzzers return whether they support/don't support the specific action. We would then even be able to keep the fuzzing loop common among all fuzzers.